sqli_token.go

```diff
@@ -23,7 +23,7 @@ const (
 
 // Look forward for doubling of delimiter
 //
-// case 'foo''bar' -> foo''bar
+// case 'foo' 'bar' -> foo' 'bar
```

Comment:
I had to add the spaces because the linter was complaining and wanted to merge `''` into `"`. Also my IDE was trying to do the same 🤔:
```
-// case 'foo''bar' -> foo''bar
+// case 'foo”bar' -> foo”bar
```